FTL.h

```diff
@@ -160,7 +160,7 @@ typedef struct {
 	int domainID;
 	int clientID;
 	int forwardID;
-	bool db;
+	sqlite3_int64 db;
```

Comment:
The SQlite3 definition itself is a bit more involved than simply using something like `long long int` because there are architectures (like the RPi 1 B+ IIRC) that do not really support any 64 byte integer architecture-wise.

See also https://www.sqlite.org/c3ref/int64.html